src/components/NavBar.jsx

```diff
@@ -1,52 +1,64 @@
-import React, { Component } from "react";
+import React from "react";
+import "../styles/navBar.css"
+import {NavLink, Link} from 'react-router-dom'
+import Logo from "./assets/logo.png"
+import MessageIcon from "./assets/message.png"
+import NotificationIcon from "./assets/notification.png"
+import ProfileIcon from "./assets/profile.png"
```

Comment:
You can get all these icons from react-icons or fontawesome libraries. Make sure your assets folder is cleaned up and only contains things like your logo and other pictures, but not icons (except you create a custom icon yourself that doesn't exist in any of the above libraries).